hack/kokoro/release.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Copyright 2021 The Skaffold Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+export GO_VERSION=1.17.3
+export GO_VERSION_SHA=765c021e372a87ce0bc58d3670ab143008dae9305a79e9fa83440425529bb636
+
+# Replace go installation with version of our choosing
+rm -Rf /usr/local/go && mkdir /usr/local/go
+curl --fail --show-error --silent --location https://go.dev/dl/go${GO_VERSION}.darwin-amd64.tar.gz -o go${GO_VERSION}.darwin-amd64.tar.gz
+if [ "$(shasum -a 256 go${GO_VERSION}.darwin-amd64.tar.gz | cut -d ' ' -f 1)" != ${GO_VERSION_SHA} ]; then
+  echo "failed to verify sha of downloaded go installation"
+  exit 1
+fi
+
+tar xz --directory=/usr/local/go --strip-components=1
+
+# Make binaries
+make -f Makefile -f hack/kokoro/Makefile kokoro-cross
```

Comment:
Doesn't this need to define SUPPORTED_PLATFORMS and PROJECT?

But do we need to have a separate makefile?  Can't we just use `make cross` and then copy `out/*` instead of `build/*`?

Reply:
we can't use make cross as it uses docker to build these binaries